pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -426,6 +432,29 @@ private void initPinotCrypterFactory() {
     }
   }
 
+  /**
+   * Register and connect to Helix cluster as PARTICIPANT role.
+   */
+  private HelixManager registerAndConnectAsHelixParticipant() {
+    String controllerParticipantInstanceId = _helixResourceManager.getInstanceId();
```

Comment:
Don't fetch instanceId from resource manager, explicitly create one instanceId for controller, one for participant.

Reply:
Done